src/hooks/useTodosContext.ts

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+import { TodosContext } from '../context/TodosContext';
+
+export const useTodosContext = () => {
+  const context = useContext(TodosContext);
+
+  if (context === undefined) {
```

Comment:
```suggestion
  if (!context) {
```
Don't compare with undefined, check if exists